CMSgov/design-system
# Refactor Autocomplete to remove utils and update snapshot and vis reg…

## Summary
[Jira 1101](https://jira.cms.gov/browse/WNMGDS-1101)

### Added
Added two new classes and refactored the utils appropriately:
- `.ds-c-autocomplete__clear-btn`
- `.ds-c-autocomplete__label`
Consolidated styles for Autocomplete list into `.ds-c-autocomplete__list`.

## How to test
Compare [test url](http://design-system-demo.s3-website-us-east-1.amazonaws.com/WNMGDS-1101/autocomplete-util-classes/components/autocomplete/) to [PROD](https://design.cms.gov/components/autocomplete/) - there should be no changes.

Our Backstop references images were outdated, therefore were unreliable to confirm no changes took place. 

New examples were added to the Autocomplete documentation that altered the layout of the entire page. Also, these VisReg images didn't capture the interactive appearance when a user enters information, so there was no way to compare the Autocomplete list view.
